README.md

```diff
@@ -1,175 +1,119 @@
-# TAMR AWS EMR Terraform Module
-This module creates the entire AWS infrastructure required for Tamr to work with AWS EMR. Currently, this module supports 3 patterns of use:
-1. Creation of infrastruction for static HBase cluster
-2. Creation of infrastructure for static Spark cluster
-3. Creation of infrastructure for ephemeral Spark cluster (the cluster itself is not created)
-
-# Examples
-## Minimal
-Fully working examples for each pattern of use. These examples might require extra resources to run the examples.
-### Invokes the root module:
-- [Static HBase Cluster](https://github.com/Datatamer/terraform-aws-emr/tree/master/examples/static-hbase)
-- [Static Spark Cluster](https://github.com/Datatamer/terraform-aws-emr/tree/master/examples/static-spark)
-### Invokes submodules:
-- [Ephemeral Spark Cluster](https://github.com/Datatamer/terraform-aws-emr/tree/master/examples/ephemeral-spark)
-
-# Resources Created
-This module creates:
-* 5 Security Groups
-    * One security group for EMR Managed Master instance(s)
-    * One security group for EMR Managed Core instance(s)
-    * One security group for additional ports for Master instance(s)
-    * One security group for additional ports for Core instance(s)
-    * One service access security group that can be attached to any instance
-* Security group rules. The number of the security group rules varies based on the number of CIDRs or source SGs provided.
-* 2 IAM Policies:
-    * Minimum required EMR service policy
-    * Minimum required EMR EC2 policy
-* 2 IAM roles:
-    * Tamr EMR service IAM role
-    * Tamr EMR EC2 IAM role
-* 1 IAM instance profile for EMR EC2 instances
-* 1 bucket object with the cluster's startup script
-
-If you are creating a static HBase or Spark cluster, this module also creates:
-* 1 EMR Cluster and associated EMR Security Configuration
-
-Note: For creating the logs and root directory buckets and/or S3-related permissions, use the [terraform-aws-s3](https://github.com/Datatamer/terraform-aws-s3) module.
-
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+<!-- BEGIN_TF_DOCS -->
```

Comment:
It looks like this changed from `<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->`	 to `<!-- BEGIN_TF_DOCS -->` in the latest docker images, which is what's wiping out the non generated parts of the doc. 

I tried replacing it with the new tag and running it locally, but it's still getting wiped out.

I'll ask about it internally, but in the meantime can you put back the docs outside of the tags?

Reply:
What do you mean by back the docs outside of the tags?